pkgs/top-level/python-packages.nix

```diff
@@ -6708,12 +6708,20 @@ in {
 
   zerobin = callPackage ../development/python-modules/zerobin { };
 
-  tensorflow-estimator = callPackage ../development/python-modules/tensorflow-estimator { };
+  tensorflow-estimator = self.tensorflow-estimator-1;
```

Comment:
I'm pretty sure self is deprecated (by you in fact!):

https://github.com/NixOS/nixpkgs/blob/728188794e6ce021948697c967318934907d67fa/pkgs/top-level/all-packages.nix#L14-L21

Reply:
@matthewbauer Ah yes, but this is `python-packages`, where I think `self` has it's normal, fine meaning. What makes `all-packages.nix` so insidious is not only is the thing called `self` bad, it is different than the thing normally called `self`.

Reply:
Oops! Makes sense